samples/msal-node-samples/auth-code-cli-brokered-app/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "msal-node-auth-code-cli-brokered-app",
+  "version": "1.0.0",
```

Comment:
Do we want to start with 0.0.1 vs 1.0.0?  1.0.0 implies we can't change any interfaces and we might want to prior to release.

Reply:
This is a sample app, versioning here doesn't really mean anything but a value is required.